tests/test_payments.py

```diff
@@ -48,14 +47,155 @@
 DC_RETURN['charge_history'] = 0.0
 DC_RETURN['payment_id'] = 2
 
+SAMPLE_PAYMENT = {
+    'id': 0,
+    'nickname': 'my credit',
+    'user_id': 1,
+    'payment_type': 'credit',
+    'details':
+        {
+            'user_name': 'Jimmy Jones',
+            'card_number': '1111222233334444',
+            'expires': '01/2019',
+            'card_type': 'Mastercard'
+        }
+}
```

Comment:
maybe we should consider pulling out all this test data and importing it just to clean up the tests a bit. not a priority, but just occurred to me. can be separate refactor issue at some point